### DISCUSSION TITLE
蓝灯mac苹果客户端更新到7.9.5后，专业账号无法登录，请帮忙处理。

### DISCUSSION THREAD

**Discussion by dongc** [getlantern/lantern-forum-cn#866] [Q&A]
蓝灯mac苹果客户端更新到7.9.5后，专业账号无法登录，请帮忙处理。

- mac 苹果：2.9 GHz 双核Intel Core i5
- 浏览器：arc
- 蓝灯客户端版本：7.9.5
- 专业账户邮箱：dongc2010@gmail.com   专业账户有效期到:   02/23/2026

![12](https://github.com/user-attachments/assets/281d5c67-efe9-4eec-9f6a-a4cf0872aaa2)

**Comment by MaxMKMax** [comment]
@dongc 您好. 请您也提供我支付单据. 我帮您提交问题上去.

**Reply by dongc** [reply]
![WechatIMG362](https://github.com/user-attachments/assets/52272aae-ee19-4b19-a728-ceee8f42d4cb)
银联支付的。
邮箱账号：dongc2010@gmail.com

@Mike777Khoo 请帮忙处理。
方法都试过了，mac 苹果客户端，无法登录专业账号。

**Reply by dongc** [reply]
mac苹果电脑端通过邮箱登录专业账号，日志内容见如下：
[lantern.log](https://github.com/user-attachments/files/17028596/lantern.log)

**Reply by dongc** [reply]
@Mike777Khoo 请你尽快联系处理。多谢🙏

**Reply by dongc** [reply]
@Mike777Khoo 你好，什么时候能解决问题。不行的话，申请退款，影响正常使用了。

**Reply by dongc** [reply]
@Mike777Khoo 什么时候，能解决。

**Reply by dongc** [reply]
@Mike777Khoo 你好
mac 苹果客户端，无法登录专业账号。这个问题，什么时候能解决！！！

**Reply by dongc** [reply]
@Mike777Khoo 你好
mac 苹果客户端，无法登录专业账号。这个问题，什么时候能解决！！！

**Reply by dongc** [reply]
@Mike777Khoo mac 苹果客户端，无法登录专业账号。这个问题，什么时候能解决！！

**Reply by dongc** [reply]
@Mike777Khoo 你好，仅有 andriod 手机，现在也无法连接 lantern，请处理。

**Comment by greySkiesMarsh** [comment]
Hello @dongc 

I'm sorry to hear that you're having trouble logging into Lantern Pro on desktop. We've identified an issue that is causing some unexpected errors with account linking. 

I can see your account in our system, and it looks like you have one (Nokia) device linked to your account. If possible, could you please try to use the "Device Linking" option to see if that works?

**Reply by dongc** [reply]
Through authorization, I can't get my Mac computer to log in to Lantern normally

**Reply by dongc** [reply]
this is log ,as follow
[lantern.log](https://github.com/user-attachments/files/17028749/lantern.log)

**Reply by dongc** [reply]
@greySkiesMarsh Mac and Apple client, unable to log in to the professional account. When will this problem be solved!!

**Reply by dongc** [reply]
@greySkiesMarsh Mac and Apple client, unable to log in to the professional account. When will this problem be solved!!

**Reply by dongc** [reply]
@greySkiesMarsh Mac and Apple client, unable to log in to the professional account. When will this problem be solved!!
